test/integration/apiserver/cel/typeresolution_test.go

```diff
@@ -354,6 +355,14 @@ func TestBuiltinResolution(t *testing.T) {
 				if gvk.Version == "__internal" {
 					continue
 				}
+				// apiextensions.k8s.io/v1beta1 not published
+				if tc.name == "discovery" && gvk.Group == "apiextensions.k8s.io" && gvk.Version == "v1beta1" {
+					continue
+				}
+				// apiextensions.k8s.io ConversionReview not published
+				if tc.name == "discovery" && gvk.Group == "apiextensions.k8s.io" && gvk.Kind == "ConversionReview" {
+					continue
+				}
```

Comment:
Would it be hard/make sense to iterate all the schemas known to the resolver rather than schemas known to the scheme? 

Im leaning that way so we wouldn't need to have exceptions for types without a schema, and could test all schemas rather than select few.

Reply:
> Would it be hard/make sense to iterate all the schemas known to the resolver 
The resolver resolves schemas but do not have the capability to enumerate schemas. Reading the list of schemas from the Scheme is the best approximation of the enumeration. And this is for testing only. At runtime there is no need for enumerating schemas anyways.